changelog/v1.17.0-beta15/distroless-all.yaml

```
@@ -0,0 +1,4 @@
+changelog:
+- type: NON_USER_FACING
+  description: Adds distroless variants to all gloo edge created images
```

Nit: I'm just trying to be precise to make git history slightly more useful
```suggestion
  description: Adds distroless variants to all remaining gloo edge created images (certgen, kubectl, accesslogger, discovery, ingress)
```